manifest.toml

```
@@ -0,0 +1,93 @@
+packaging_format = 2
+
+id = "framaforms"
+name = "Framaforms"
+description.en = "Create online webforms and surveys"
+description.fr = "Créez des formulaires et questionnaires en ligne"
+
+version = "1.1~ynh1"
```

```suggestion
version = "1.0.3~ynh1"
```

inconsistent version compared to sources below ...